client.go

```diff
@@ -181,6 +183,94 @@ func (c *Client) Schema() (*Schema, error) {
 	return schema, nil
 }
 
+// ImportFrame imports bits from the given CSV iterator
+func (c *Client) ImportFrame(frame *Frame, bitIterator BitIterator, batchSize uint) error {
+	const sliceWidth = 1048576
+	linesLeft := true
+	bitGroup := map[uint64][]Bit{}
+	var currentBatchSize uint
+	indexName := frame.index.name
+	frameName := frame.name
+
+	for linesLeft {
+		bit, err := bitIterator.NextBit()
```

Comment:
I got the inspiration from: https://godoc.org/google.golang.org/api/iterator . They named the iterator `NextPage`.